test/integration/api.bats

```diff
@@ -207,9 +242,19 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker start" {
-	skip
+	start_docker 3 
+	swarm_manage
+	# create
+	run docker_swarm create --name test_container busybox sleep 1000
+	[ "$status" -eq 0 ]
+
```

Comment:
After creating the container, the status of the new container is null. I will check the lines of "docker ps -l" and check the container's name